### ISSUE TITLE
Minio console is assigned a dynamic port instead of 9001 in distributed mode

### ISSUE THREAD

**Issue by mohanlon-iai** [bitnami/charts#7527]
**Which chart**:
Minio version 8.0.0

**Describe the bug**
When deploying Minio in distributed mode, the port for the console is dynamically allocated like a nodeport would be. I use Skaffold to deploy the Minio Helm chart to Kubernetes in Windows 10 Docker Desktop (4.x).

**To Reproduce**

1. Deploy in distributed mode

**Expected behavior**
The console port should be 9001. 

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
$ helm version
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"clean", GoVersion:"go1.14.11"}
```

- Output of `kubectl version`:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.18-dispatcher", GitCommit:"de944d802c49735ad0f4bfe82ddfa19737ebe962", GitTreeState:"clean", BuildDate:"2021-05-13T17:56:14Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.4", GitCommit:"3cce4a82b44f032d0cd1a1790e6d2f5a55d20aae", GitTreeState:"clean", BuildDate:"2021-08-11T18:10:22Z", GoVersion:"go1.16.7", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
![image](https://user-images.githubusercontent.com/33632874/133792075-99d00561-b9b3-435e-a14b-02b907bcacf0.png)
[minio-0-logs.txt](https://github.com/bitnami/charts/files/7186138/minio-0-logs.txt)

**Comment by juan131** [comment]
Hi @mohanlon-iai 

I'm not sure if it's related but we recently released a new version of the chart (`8.0.1`) fixing an issue related to the service configuration for the API port, see: 

- https://github.com/bitnami/charts/commit/ce6c343f2b03497863e06597ec8199b37b22dde6#diff-2712fd2badfcf92768d33353a1d5c225cf92f9e20895448b743251fe1c962417

Could you please give it a try?

--- 

By the way, the ports for MinIO® Console and MinIO® API at container level are customizable using the `containerPorts.api` and `containerPorts.console` parameters as it's mentioned in the link below:

- https://github.com/bitnami/charts/tree/master/bitnami/minio#minio-deploymentstatefulset-parameters

While you can customize them at service level using `service.ports.api` and `service.ports.console` parameters:

- https://github.com/bitnami/charts/tree/master/bitnami/minio#traffic-exposure-parameters

**Comment by ondrejmo** [comment]
> Hi @mohanlon-iai
> 
> I'm not sure if it's related but we recently released a new version of the chart (`8.0.1`) fixing an issue related to the service configuration for the API port, see:
> 
>     * [ce6c343#diff-2712fd2badfcf92768d33353a1d5c225cf92f9e20895448b743251fe1c962417](https://github.com/bitnami/charts/commit/ce6c343f2b03497863e06597ec8199b37b22dde6#diff-2712fd2badfcf92768d33353a1d5c225cf92f9e20895448b743251fe1c962417)
> 
> 
> Could you please give it a try?
> 
> By the way, the ports for MinIO® Console and MinIO® API at container level are customizable using the `containerPorts.api` and `containerPorts.console` parameters as it's mentioned in the link below:
> 
>     * https://github.com/bitnami/charts/tree/master/bitnami/minio#minio-deploymentstatefulset-parameters
> 
> 
> While you can customize them at service level using `service.ports.api` and `service.ports.console` parameters:
> 
>     * https://github.com/bitnami/charts/tree/master/bitnami/minio#traffic-exposure-parameters

The bug is still present in version `8.0.1`, the problem seems to be in the image rather than the chart, since even when env variable  `MINIO_CONSOLE_PORT_NUMBER=9001` is added via `extraEnv` directive, minio console still binds to a dynamic port.

**Comment by mohanlon-iai** [comment]
@juan131 I tried both and it still assigned a dynamic port to the console. Sorry! I should have mentioned that when I wrote up the issue.

**Comment by mohanlon-iai** [comment]
@ondrejmo Good find on the extraEnv! I didn't think to try that. I will try that next time I'm troubleshooting problems with a chart.

**Comment by juan131** [comment]
> even when env variable `MINIO_CONSOLE_PORT_NUMBER=9001` is added via `extraEnvs` directive

@ondrejmo you won't need to use `extraEnvs` parameter for this anymore thanks to the changes introduced in [this PR](https://github.com/bitnami/charts/pull/7526/files) (version `8.1.0`)

> minio console still binds to a dynamic port

That's weird since the port should be static and based on that env. variable as you can see [in this line off the code](https://github.com/bitnami/bitnami-docker-minio/blob/master/2021/debian-10/rootfs/opt/bitnami/scripts/minio/run.sh#L20). I'll share this with the colleague that has been working on the recent changes in the image/chart.

**Comment by juan131** [comment]
My colleague just told me the issue should be solved thanks to [this contribution](https://github.com/bitnami/bitnami-docker-minio/pull/24), could you give a try to the latest chart version?

**Comment by mohanlon-iai** [comment]
It looks like the updates worked for me. Thanks!

**Comment by juan131** [comment]
Great! I proceed to close this issue since it seems it's solved in version `8.1.0`. Feel free to reopen it if you still face it.
